pkg/runtest/run.go

```diff
@@ -305,13 +305,11 @@ func parseProg(target *prog.Target, dir, filename string, requires map[string]bo
 	if err != nil {
 		return nil, nil, nil, fmt.Errorf("failed to read %v: %w", filename, err)
 	}
-	properties := parseRequires(data)
```

Comment:
There was one more reason to split `parseRequires` and `ParseSeed()`: avoid wasting CPU time on parsing irrelevant seeds. E.g. all `syz_mount_image` seed files have the `manual` tag and thus we do not need to parse and decompress all of them during tests.

I think we'd better preserve this behavior, so it means we either
* Export `ParseRequires` from `pkg/manager` and keep the logic here as it was.
* Make `ParseSeedStrict` take the `requires` argument and do the necessary checks inside (possibly rename `ParseSeedStrict` to something more suitable as well).

Reply:
Done (added an extra parameter to ParseSeedStrict and renamed it)